hackathon_site/dashboard/frontend/src/components/teamDetail/TeamPendingOrderTable/TeamPendingOrderTable.test.tsx

```diff
@@ -11,40 +11,51 @@ import {
 import {
     mockCartItems,
     mockHardware,
+    mockHardwareWithBuffer,
     mockPendingOrdersInTable,
 } from "testing/mockData";
 import TeamPendingOrderTable from "components/teamDetail/TeamPendingOrderTable/TeamPendingOrderTable";
 
 describe("team pending order table", () => {
     test("renders team pending order table", () => {
-        const { getByTestId } = render(<TeamPendingOrderTable />);
+        const { getByTestId } = render(
+            <TeamPendingOrderTable
+                orders={mockPendingOrdersInTable}
```

Comment:
ah I see why you want these props for tests. So generally we don't usually change our code to help our tests, we write code in the best way possible and there's usually always a way to write tests around it.

For here, what you'll need to do is mock the redux store (you can search around in the Dashboard, ItemTable or Inventory page tests for some examples) but essentially you need to use the `mockStoreWithEntities()` function from `testing/utils.tsx` (so you can remove these 2 props)